pipelines/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/DataDog/dd-trace-go/pipelines
```

Comment:
Why is this its own module? Happy to discuss this point offline too.

Reply:
I want to be able to deploy that module without updating the tracer.
+ this module will have its API change, so it's in alpha mode. But dd-trace-go is released under a stable API.